### DISCUSSION TITLE
GAMES101 - CS自学指南

### DISCUSSION THREAD

**Discussion by giscus** [PKUFlyingPig/cs-self-learning#126] [Announcements]
# GAMES101 - CS自学指南

CS自学指南

https://csdiy.wiki/%E8%AE%A1%E7%AE%97%E6%9C%BA%E5%9B%BE%E5%BD%A2%E5%AD%A6/GAMES101/

**Comment by TsuiMinghung** [comment]
请问有纯粹关于C programing的课程吗，我看基本都是C++的

**Reply by ghost** [reply]
浙大翁恺的就不错

**Reply by wmasfoe** [reply]
https://www.bilibili.com/video/BV1sJ411E7St/

**Reply by TsuiMinghung** [reply]
翁凯的有点太简单了，我已经大二了，写操作系统的时候发现C基础不够好，需要稍微有一定难度的C课程来巩固

**Reply by Saltedf** [reply]
https://www.bilibili.com/video/BV18p4y167Md 
https://book.douban.com/subject/6025290/

**Reply by Value-Jack** [reply]
写操作系统的话可以看看清华方勇的操作系统，里面有对应的lab

**Comment by guoxingyu301** [comment]
cs61A没有学校账号是不是看不了recording？

**Reply by linengcs** [reply]
往年的可以 好像18年之后就只有学生可以看了 你可以翻翻

**Comment by Episode-Zhang** [comment]
games101的官网是不是打不开了，我这边显示“数据库连接错误”，搜了一圈没找到相关的结果

**Reply by Episode-Zhang** [reply]
应该是暂时性的，现在已经可以打开了！

**Comment by Weinberg17** [comment]
可以问问有关于Qt的教程吗？非科班只有数学基础，自学CG，要被UI鼠标交互给弄疯了。。

**Reply by WXYSW** [reply]
请问您现在有没有推荐的教程

**Comment by MolinDeng** [comment]
个人的学习仓库，会更新作业，欢迎参考 https://github.com/MolinDeng/games101

**Comment by linearalgebrayhz** [comment]
寒假里一天两集+一天一作业做完了GAMES101，图形学的方方面面真的非常令人着迷。闫老师深入浅出的讲解即便对于缺乏基础的同学也很友好，衷心感谢闫老师等一众图形学布道者

**Reply by goxiang** [reply]
请问在哪里看的视频呢？

**Reply by linearalgebrayhz** [reply]
抱歉回复的太晚了，课程在bilibili上面都有的 只需要搜索GAMES101 计算机图形学入门

**Comment by potateosssssssss** [comment]
课程资源在博主GitHub哪个部分里啊？没找见呢。话说资源里有虚拟网盘吗？清华云盘里找不到
